src/app/App.js

```diff
@@ -1,4 +1,7 @@
+import answearDisplay from '../answearDisplay'
```

Comment:
You mean 
import answearDisplay from '../answearDisplay/answearDisplay' 
?

Reply:
yes ofcourse, I will change that.